src/aind_metadata_service/sharepoint/nsb2023/mapping.py

```diff
@@ -1264,70 +3497,474 @@ def aind_burr_6_hemisphere(self) -> Optional[Side]:
+                self._nsb.burr_6_intended_x0021.VOLTAGE: None,
+                self._nsb.burr_6_intended_x0021.CONTROL: None,
+            }.get(self._nsb.burr_6_intended_x0021, None)
+        )
+
+    @property
+    def aind_burr_6_intended_x0022(self) -> Optional[Any]:
+        """Maps burr_6_intended_x0022 to aind model."""
+        return (
+            None
+            if self._nsb.burr_6_intended_x0022 is None
+            else {
+                self._nsb.burr_6_intended_x0022.N_A: None,
+                self._nsb.burr_6_intended_x0022.DOPAMINE: None,
+                self._nsb.burr_6_intended_x0022.SEROTONIN: None,
+                self._nsb.burr_6_intended_x0022.NOREPINEPHRINE: None,
+                self._nsb.burr_6_intended_x0022.ACETYLCHOLINE: None,
+                self._nsb.burr_6_intended_x0022.HISTAMINE: None,
+                self._nsb.burr_6_intended_x0022.GLUTAMATE: None,
+                self._nsb.burr_6_intended_x0022.GABA: None,
+                self._nsb.burr_6_intended_x0022.CALCIUM: None,
+                self._nsb.burr_6_intended_x0022.VOLTAGE: None,
+                self._nsb.burr_6_intended_x0022.CONTROL: None,
+            }.get(self._nsb.burr_6_intended_x0022, None)
+        )
+
+    @property
+    def aind_burr_6_intended_x0023(self) -> Optional[Any]:
+        """Maps burr_6_intended_x0023 to aind model."""
+        return (
+            None
+            if self._nsb.burr_6_intended_x0023 is None
+            else {
+                self._nsb.burr_6_intended_x0023.N_A: None,
+                self._nsb.burr_6_intended_x0023.DOPAMINE: None,
+                self._nsb.burr_6_intended_x0023.SEROTONIN: None,
+                self._nsb.burr_6_intended_x0023.NOREPINEPHRINE: None,
+                self._nsb.burr_6_intended_x0023.ACETYLCHOLINE: None,
+                self._nsb.burr_6_intended_x0023.HISTAMINE: None,
+                self._nsb.burr_6_intended_x0023.GLUTAMATE: None,
+                self._nsb.burr_6_intended_x0023.GABA: None,
+                self._nsb.burr_6_intended_x0023.CALCIUM: None,
+                self._nsb.burr_6_intended_x0023.VOLTAGE: None,
+                self._nsb.burr_6_intended_x0023.CONTROL: None,
+            }.get(self._nsb.burr_6_intended_x0023, None)
+        )
+
     @property
     def aind_burr_6_m_l(self) -> Optional[Decimal]:
         """Maps burr_6_m_l to aind model"""
         return self._map_float_to_decimal(self._nsb.burr_6_m_l)
 
     @property
     def aind_burr_hole_1(self) -> Optional[BurrHoleProcedure]:
-        """Maps burr_hole_1 to aind model"""
+        """Maps burr_hole_1 to aind model."""
         return (
             None
             if self._nsb.burr_hole_1 is None
             else {
                 self._nsb.burr_hole_1.SELECT: None,
+                self._nsb.burr_hole_1.STEREOTAXIC_INJECTION: BurrHoleProcedure.INJECTION,
+                self._nsb.burr_hole_1.SPINAL_INJECTION: BurrHoleProcedure.INJECTION,
```

Comment:
Are we combining spinal injections in with brain injections?

Reply:
That's a good catch. Right now the spinal injections info is being stored in burr hole info (which yes is default being mapped to brain injections if it's not specified as Ionto or Nano). How should spinal and grid injections be mapped?